schematron/v2.2.0/v2-2-0_L200_Audit.csv

```diff
@@ -230,7 +230,7 @@
 "","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Systems/auc:PumpSystems/auc:PumpSystem","false","","","auc:PumpControlType","","",""
 "HVAC Distribution System Delivery Outdoor Air Control","ASHRAE 211 6.2.1.3 (c)","Central Air Distribution","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Systems/auc:HVACSystems/auc:HVACSystem/auc:HeatingAndCoolingSystems/auc:Deliveries/auc:Delivery/auc:DeliveryType/auc:CentralAirDistribution","false","","","auc:FanBased/auc:AirSideEconomizer","","",""
 "","","","","","/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Systems/auc:HVACSystems/auc:HVACSystem/auc:HeatingAndCoolingSystems/auc:Deliveries/auc:Delivery/auc:DeliveryType/auc:CentralAirDistribution/auc:FanBased/auc:AirSideEconomizer","false","","","auc:AirSideEconomizerType","","",""
-"","","","","","","","","","auc:AirSideEconomizerType != 'None' or auc:EconomizerControl","","",""
+"","","","","","","","","","auc:AirSideEconomizerType = 'None' or auc:EconomizerControl","","",""
```

Comment:
Hey Ted, can we make this more explicit:
1. If the `auc:AirSideEconomizerType = 'None'`, then we don't want an `auc:EconomizerControl`
2. If it is not None, then we must have the `auc:EconomizerControl`

Basically, an exclusive or.

Reply:
Addressed here: baedd9e4b1655c914e33b95ae06aea9c6b36818d